uber/templates/signups/shifts.html

```diff
@@ -144,8 +142,9 @@
 
                 buttonTag += 'onclick="click_shift(\'' + event.taken;
                 buttonTag += '\',\'' +  event.id;
-                                buttonTag += '\',\'' +  moment(event.start).toISOString();
-                                buttonTag += '\',\'' +  $('<div/>').text(event.title).html();
+                                buttonTag += '\',\'' + moment(event.start).format("h:mma");
```

Comment:
The confirmation dialogue was displaying an ISO 8601 string, so it would ask people if they wanted to drop a shift at, e.g., "2017-07-31T13:01:20+00:00." I made it prettier.